src/common/sqonUtils.js

```diff
@@ -221,3 +232,35 @@ export const addSetToActiveQuery = ({ setId, querySqons, activeIndex }) => {
   newSqons[activeIndex] = activeSqon;
   return newSqons;
 };
+
+export const addFieldToActiveQuery = ({ term, querySqons, activeIndex }) => {
+  if (!term) {
```

Comment:
I would remove that check si it's the caller responsibility to pass correct term.